### PR TITLE
Feature/indy v1.6.4

### PR DESCRIPTION
**Pull request by alkopnin**
Migration on Indy v1.6.4

### CODE REVIEW

File: genesis/docker_pool_transactions_genesis.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{"reqSignature":{},"txn":{"data":{"data":{"alias":"Node1","blskey":"4N8aUNHSgjQVgkpm8nhNEfDf6txHznoYREg9kirmJrkivgL4oSEimFF6nsQ6M41QvhM2Z33nves5vfSn9n1UwNFJBYtWVnHYMATn76vLuL3zU88KyeAYcHfsih3He6UHcXDxcaecHVz6jhCYz1P2UZn2bDVruL5wXpehgBfBaLKm3Ba","client_ip":"127.0.0.1","client_port":9702,"node_ip":"127.0.0.1","node_port":9701,"services":["VALIDATOR"]},"dest":"Gw6pDLhcBcoQesN72qfotTgFa7cbuqZpkX3Xo6pLhPhv"},"metadata":{"from":"Th7MpTaRZVRYnPiabds81Y"},"type":"0"},"txnMetadata":{"seqNo":1,"txnId":"fea82e10e894419fe2bea7d96296a6d46f50f93f9eeda954ec461b2ed2950b62"},"ver":"1"}
```
Comment (voddan) [inline genesis/docker_pool_transactions_genesis.txt]: What changed in this file? Why?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +0,0 @@
-#Thu Jul 19 15:03:45 MSK 2018
```
Comment (voddan) [inline gradle/wrapper/gradle-wrapper.properties]: Any gradle project must have the files in `/gradle/wrapper` have repeatable builds. Please revert those files back.

File: genesis/docker_pool_transactions_genesis.txt (hunk 1 above)
Comment (alkopnin) [reply genesis/docker_pool_transactions_genesis.txt]: Additional value was added

Comment (voddan) [reply genesis/docker_pool_transactions_genesis.txt]: Ok, i see. Thanks!

**Comment by voddan** [comment]
So the tests needed fixing, huh? :D 
I am still a bit worried that your setup does not allow you to run automated `gradle build`. How much time will it take for you to adapt the gradle build for MacOS? Do you think you could include it in this PR?

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (alkopnin) [reply gradle/wrapper/gradle-wrapper.properties]: reverted. Thanks

**Pull request merged**